plugins/evaluation-plugin/core/src/com/intellij/cce/processor/CallCompletionProcessor.kt

```diff
@@ -57,13 +58,19 @@ class CallCompletionProcessor(private val text: String,
       var currentPrefix = ""
       if (prefixCreator.completePrevious) {
         for (symbol in prefix) {
-          addAction(CallCompletion(currentPrefix, token.text, token.properties))
+          if (token.properties.tokenType == TypeProperty.VARIABLE_DECLARATION)
+            addAction(CallRename(currentPrefix, token.text, token.properties))
```

Comment:
I thought about this more, and it seems that we're not so interested in metrics depending on different prefixes of the target name. When we want to rename variable, we use this feature just invoking it on the existing name. So let's replace target name with some specified token and invoke rename on it once instead of using different prefixes.